runtime-library/libdyntype/dynamic-simple/dyn-value/class/dyn_class.c

```diff
@@ -98,7 +98,7 @@ find_class_constructor(const char *name)
 
     for (i = 0; i < DynClassEnd; i++) {
         meta = class_meta_array[i];
-        if (!meta)
+        if (!meta || !meta->name)
```

Comment:
If the name field is necessary, we should `assert(meta->name != NULL)` at before L104, and we should add name for all class_metas

Reply:
Done